src/main/java/duke/command/Duke.java

```diff
@@ -0,0 +1,277 @@
+package duke.command;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.ToDo;
+
+import java.util.ArrayList;
+import java.io.File;
+import java.io.IOException;
+import java.util.Scanner;
+import java.io.FileWriter;
+
+public class Duke {
+    public static final String TODO = "todo";
+    public static final String DEADLINE = "deadline";
+    public static final String EVENT = "event";
+    public static final String LIST = "list";
+    public static final String DONE = "done";
+    public static final String EXIT = "bye";
+    public static final String DELETE = "delete";
+    private static final ArrayList<Task> taskList = new ArrayList<>();
+    private static int numberOfTasks = 0;
+    private static File save;
+
+    public static void main(String[] args) {
+        saveCheck();
+        greet();
+        instructions();
+        chooseTask();
+    }
+
+    private static void saveCheck() {
+        try {
+            File directory = new File("data");
+            if (!directory.exists()) {
+                directory.mkdir();
+            }
+            save = new File(directory + "/" + "duke1.txt");
+            save.createNewFile();
+            Scanner s = new Scanner(save); // create a Scanner using the File as the source
+            int counter = 1;
+            while (s.hasNext()) {
+                String[] parts = s.nextLine().split("\\|");
+                String taskType = parts[0].trim();
+                String status = parts[1].trim();
+                String taskDescription = parts[2].trim();
+                switch (taskType) {
+                case "T":
+                    addTask(TODO, taskDescription);
+                    break;
+                case "D":
+                    addTask(DEADLINE, taskDescription);
+                    break;
+                case "E":
+                    addTask(EVENT, taskDescription);
+                    break;
+                }
+                if (status.equals("X")) {
+                    markTaskComplete(Integer.toString(counter));
+                }
+                counter++;
+            }
+
+        } catch (IOException e) {
+            System.out.println("Something went wrong: " + e.getMessage());
+        }
+    }
+
+    private static void writeToFile() throws IOException {
+        FileWriter fw = new FileWriter(save);
+        for (int i = 0; i < numberOfTasks; i++) {
+            fw.write(taskList.get(i).getType() + " | " + taskList.get(i).getStatusIcon() + " | " + taskList.get(i).getOriginalDescription() + System.lineSeparator());
```

Comment:
This line exceeds the standard of [<120 characters per line](https://se-education.org/guides/conventions/java/basic.html#layout).  Please check other files as there are similar lines like this. 

You could try formatting strings using [Java String format()](https://www.javatpoint.com/java-string-format)  and assigning variable names to the parameters.